### ISSUE TITLE
Alt Text on Cell output

### ISSUE THREAD

**Issue by whitead** [whitead/dmol-book#6]
Many figures are output without alt text. This should be fixed.

**Comment by whitead** [comment]
Decided cell output is clearly connected to code, so not necessary
